components/osol/ddu/Makefile

```diff
@@ -28,12 +28,19 @@ include $(WS_MAKE_RULES)/ips.mk
 
 PCI_IDS_FILE=/usr/share/pci.ids.gz
 
+# Clean out the BUILD_DIR at the same time if the Makefile is different
+# otherwise the packaging won't believe that it should rebuild the pkg
+# because the files there are the same.
+$(SOURCE_DIR)/.prep: $(MAKEFILE_PREREQ)
+	$(TOUCH) $@
+	$(RM) -rf $(BUILD_DIR)
+
 build:
 
 # Using gzipped pci.ids at another location than /usr/ddu/data/ is not possible
 # until all_devices is modified. For now just copy the file as expected.
 
-install:
+install: build $(SOURCE_DIR)/.prep
```

Comment:
Fixed in https://github.com/OpenIndiana/oi-userland/pull/2535/commits/b6ddeeb3e45e22c6730bb08005c804fa956d3845